app/models/user.rb

```diff
@@ -364,6 +364,14 @@ def set_current_taxonomies
     TopbarSweeper.expire_cache(self)
   end
 
+  def my_organizations
```

Comment:
Aren't these methods nearly equivalent to `location_and_child_ids` and `organization_and_child_ids`? As far as I can see they return the object and the others the ids, but the id is enough in the query the validator does